unblob/logging.py

```diff
@@ -2,22 +2,45 @@
 from pathlib import Path
 
 import structlog
+from dissect.cstruct import cstruct, dumpstruct
 
 
 def format_hex(value: int):
     return f"0x{value:x}"
 
 
-def pretty_print_paths(extract_root: Path):
-    def add_path(logger, method_name: str, event_dict: structlog.types.EventDict):
+def pretty_print_types(extract_root: Path):
+    def convert_type(logger, method_name: str, event_dict: structlog.types.EventDict):
         use_absolute_path = event_dict.pop("_absolute_path", False)
         for key, value in event_dict.items():
             if isinstance(value, Path):
                 path = value if use_absolute_path else value.relative_to(extract_root)
                 event_dict[key] = str(path)
+
+            elif isinstance(value, cstruct):
+                event_dict[key] = dumpstruct(value)
+
         return event_dict
 
-    return add_path
+    return convert_type
+
+
+class LazyField:
```

Comment:
I believe this (commit: 97f5700fea3f4e4805f0c9f9e77e2ac2e745f54a) was left here by mistake?

Reply:
I wanted to keep this in the history and reverted this commit.

Reply:
But you reverted the reverted commit, and it is in there, right now, and it should not be.